source/process/guest-article.rst

```diff
@@ -0,0 +1,21 @@
+Guest Articles for Mattermost Apps and Services
```

Comment:
Oh I get it know. It's an article for apps and services built for Mattermost.

E.g. Matterhorn, which is a terminal client for Mattermost.

That's okay to keep then.